localization/pose_estimator_arbiter/CMakeLists.txt

```
@@ -4,29 +4,23 @@ project(pose_estimator_arbiter)
 find_package(autoware_cmake REQUIRED)
 autoware_package()
 
-find_package(glog REQUIRED)
-
 find_package(PCL REQUIRED COMPONENTS common)
 include_directories(SYSTEM ${PCL_INCLUDE_DIRS})
 
-# ==============================
-# switch rule library
-ament_auto_add_library(switch_rule
-  SHARED
-  src/pose_estimator_arbiter/switch_rule/enable_all_rule.cpp
-)
-target_include_directories(switch_rule PUBLIC src)
-
 # ==============================
 # pose estimator arbiter node
-ament_auto_add_executable(${PROJECT_NAME}
+ament_auto_add_library(${PROJECT_NAME} SHARED
   src/pose_estimator_arbiter/pose_estimator_arbiter_core.cpp
-  src/pose_estimator_arbiter/pose_estimator_arbiter_node.cpp
+  src/pose_estimator_arbiter/switch_rule/enable_all_rule.cpp
 )
 target_include_directories(${PROJECT_NAME} PUBLIC src)
-target_link_libraries(${PROJECT_NAME} switch_rule glog::glog)
 
-# ==============================
+rclcpp_components_register_node(${PROJECT_NAME}
+  PLUGIN "PoseEstimatorArbiter"
```

@a-maumau Please try to replace
```suggestion
  PLUGIN "pose_estimator_arbiter::PoseEstimatorArbiter"
```

---

In my environment, the above suggestion has fixed all issues which you encountered.

---

Thank you very mush for checking out the codes.
I also confirmed the above suggestion works.